<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>caculator</class>
 <widget class="QMainWindow" name="caculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::PreventContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,5,6" columnstretch="5,2">
    <item row="0" column="0">
     <widget class="QWidget" name="widget_3" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="font">
          <font>
           <pointsize>52</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border:none</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="font">
          <font>
           <pointsize>52</pointsize>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">border:none</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" rowspan="2">
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="3">
        <widget class="key" name="label_14">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>cos</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="key" name="label_18">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>9</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="key" name="label_16">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>7</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="key" name="label_5">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>1</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="key" name="label_8">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>1</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="key" name="label_26">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>1</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="key" name="label_29">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>+</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="key" name="label_20">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>AC</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="key" name="label_15">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>tan</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="key" name="label_21">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>4</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="key" name="label_7">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>1</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="4">
        <widget class="key" name="label_25">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>/</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="key" name="label_27">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>2</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="key" name="label_13">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>sin</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="key" name="label_28">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>3</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="4">
        <widget class="key" name="label_30">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>-</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="key" name="label_22">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>5</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="key" name="label_9">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>log</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="key" name="label_32">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="key" name="label_31">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="key" name="label_11">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>(</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="key" name="label_10">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>ln</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="key" name="label_19">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>DEL</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="key" name="label_23">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>6</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="key" name="label_6">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="text">
          <string>1</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="key" name="label_12">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="key" name="label_2">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>ALPHA</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="key" name="label_4">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>∫</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="6" column="4">
        <widget class="key" name="label_35">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>=</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="key" name="label_17">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>8</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="key" name="label_3">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>x</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="key" name="label_24">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>x</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="key" name="label_33">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>pi</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="6" column="3">
        <widget class="key" name="label_34">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>%</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="key" name="label_36">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>ALPHA</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1" rowspan="3">
     <widget class="QWidget" name="widget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1500</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>key</class>
   <extends>QLabel</extends>
   <header location="global">key.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
